src/browser/modules/D3Visualization/lib/visualization/renders/menu.ts

```diff
@@ -157,60 +164,58 @@ const createMenuItem = function(
 }
 
 const donutRemoveNode = new Renderer({
-  onGraphChange(selection: any, viz: any) {
+  onGraphChange(selection: d3.Selection<NodeVisualisationModel>, viz: any) {
     return createMenuItem(
       selection,
       viz,
       'nodeClose',
       1,
-      'remove_node',
+      'remove-node',
       [-8, 0],
       'Remove',
       'Dismiss'
     )
   },
 
-  onTick: noop
+  onTick: noOp
 })
 
 const donutExpandNode = new Renderer({
-  onGraphChange(selection: any, viz: any) {
+  onGraphChange(selection: d3.Selection<NodeVisualisationModel>, viz: any) {
     return createMenuItem(
       selection,
       viz,
       'nodeDblClicked',
       2,
-      'expand_node',
```

Comment:
Why does this class name need to change? It's referred to in our style sheets, it'll need to be changed there as well

Reply:
We'll merge and then send a followup PR